docs/en_us/internal/testing.rst

```diff
@@ -506,23 +563,21 @@ according to the template string
 ``{scenario_number}__{step_number}__{step_function_name}__{"1_before"|"2_after"}``.
 
 If you don't want to have screenshots be captured for all steps, but
-rather want fine grained control, you can use the decorator
+rather want fine grained control, you can use this decorator before any Python function in ``feature_name.py`` file.
 
 ::
 
     @capture_screenshot_before_after
 
-before any Python function in ``feature_name.py`` file. The decorator
-will capture two screenshots - one before the decorated function runs,
-and one after. Also, the function
+The decorator will capture two screenshots - one before the decorated function runs,
```

Comment:
instead of ``` - ```, please use a colon (:) 
The decorator captures two screenshots: one before the decorated function runs,...